### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Malik02** [spotify2tidal/spotify_to_tidal#96] [General]
I'm a real noob i'm sure this very easy but i can't figure out how to use.

When i run in terminal : spotify_to_tidal
i got : bash: spotify_to_tidal: command not found

**Comment by timrae** [comment]
Did you install?

```
python3 -m pip install -e .
```

If `spotify_to_tidal` doesn't work, you can also try

```
python3 -m spotify_to_tidal
```
